### PR TITLE
Bumps Calamari.* to 20.4.2, Sashimi.* to 14.1.3, and Sashimi.AzureScripting to 14.1.0

### PR DESCRIPTION
**Pull request by mjhilton**
# Overview
This PR is one of a set of PRs to propagate a change to Calamari.Common through to Octopus Server 2021.3

Upstream changes:
* OctopusDeploy/Calamari#815
* OctopusDeploy/Sashimi#134
* OctopusDeploy/Sashimi.AzureScripting#20

Issues:
* OctopusDeploy/Issues/issues/6794
* OctopusDeploy/Issues/issues/7177

### CODE REVIEW

**Review by shaun-od — APPROVED** [review]
LGTM

File: source/Sashimi/Sashimi.csproj [hunk 1]
```diff
@@ -23,10 +23,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Octopus.Configuration" Version="4.0.0" />
```
Comment (shaun-od) [inline source/Sashimi/Sashimi.csproj:25]: actually, should we update this to 4.1.0. As we have updated it in this PR https://github.com/OctopusDeploy/Sashimi.AzureWebApp/pull/25/files

Comment (mjhilton) [reply source/Sashimi/Sashimi.csproj:25]: In that PR, the bump was from `3.0.0` to `4.1.0`, necessitated by the `net5.0` upgrade (v3 of the Octopus.Configuration library didn't have an appropriate build target to use in a `net5.0` project... I think v3 only supported `netfx`).

I'm generally being as conservative as possible on the surface area of changes and version bumps... with exception, I guess, of the Server.Extensibility ones, which had functional changes and was required to get tests passing.

Comment (shaun-od) [reply source/Sashimi/Sashimi.csproj:25]: Ok sounds reasonable.

**Pull request merged**